Tests/NIOCoreTests/AsyncSequences/NIOAsyncWriterTests.swift

```diff
@@ -13,7 +13,7 @@
 //===----------------------------------------------------------------------===//
 
 import DequeModule
-import NIOCore
+@testable import NIOCore
```

Comment:
I prefer if we don't use `@testable`. It is a horrible compiler hack and I would love to slowly get away from it by either using `SPI` or the new `package` modifier. For now let's use `@_spi(Testing)` and we can migrate that in the future.